src/server/handlers/users/{ID}/roles.ts

```diff
@@ -0,0 +1,93 @@
+/*****
+ License
+ --------------
+ Copyright © 2020 Mojaloop Foundation
+ The Mojaloop files are made available by the Mojaloop Foundation under the
+ Apache License, Version 2.0 (the 'License') and you may not use these files
+ except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop files
+ are distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ - Kevin Leyow <kevin.leyow@modusbox.com>
+
+ --------------
+ ******/
+
+import { StateResponseToolkit } from '~/server/plugins/state'
+import { Request, ResponseObject } from '@hapi/hapi'
+import { PatchDelta } from '@ory/keto-client'
+
+interface PatchOperation {
+  roleId: string;
+  action: 'insert' | 'delete';
+}
+interface UserIDRolesPatchRequest {
+  roleOperations: PatchOperation[];
+}
+
+const get = async (_context: unknown, request: Request, h: StateResponseToolkit): Promise<ResponseObject> => {
+  try {
+    const userId = request.params.ID
+    const response = await h.getKetoReadApi().getRelationTuples(
+      'role',
+      undefined,
+      'member',
+      userId
+    )
+    const rolesIdList = response.data.relation_tuples?.map(function (relationTuple) {
+      return relationTuple.object
+    })
```

Comment:
```suggestion
    const rolesIdList = response.data.relation_tuples?.map(({object}) => object)
```

Reply:
this is a result of divergance in eslint config accross projects...and my function autocomplete
I wonder if there is a proper master config file somewhere so I can stop battling linting...

I've changed to arrow functions here and throughout for other comments